src/app/(with-header-sidebar)/dashboard/[id]/page.tsx

```diff
@@ -1,3 +1,197 @@
-export default function Page() {
-  return <div>dashboard page</div>;
+'use client';
+
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'next/navigation';
+import axiosInstance from '@/lib/axiosInstance';
+import { COLUMN_URL, CARD_URL } from '@/constants/urls';
+
+import {
+  DragDropContext,
+  Droppable,
+  Draggable,
+  DropResult,
+} from 'react-beautiful-dnd';
+import styles from './page.module.css';
+
+interface Column {
+  id: number;
+  title: string;
+  teamId?: string;
+  createdAt?: string;
+  updatedAt?: string;
+  items: Card[];
+}
+
+interface Assignee {
+  profileImageUrl: string;
+  nickname: string;
+  id: number;
+}
+
+interface Card {
+  id: number;
+  title: string;
+  description: string;
+  tags: string[];
+  dueDate: string;
+  assignee: Assignee;
+  imageUrl: string;
+  teamId: string;
+  columnId: number;
+  createdAt: string;
+  updatedAt: string;
+}
+
+// interface CardResponse {
+//   cursorId: number;
+//   totalCount: number;
+//   cards: Card[];
+// }
+
+export default function DashBoardView() {
+  const { id } = useParams();
+  const [columns, setColumns] = useState<Column[]>([]);
+  // const [cards, setCards] = useState<Card[]>([]);
+  const [loading, setLoading] = useState(false);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```
해당 변수가 Boolean 값임을 명시적으로 표현하기 위해 
이런 형식을 추천드립니다.